domain-models-api-aspects/src/main/java/org/folio/rest/validators/AbstractCollectionValidator.java

```diff
@@ -0,0 +1,30 @@
+package org.folio.rest.validators;
+
+import java.lang.annotation.Annotation;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.Optional;
+
+import javax.validation.ConstraintValidatorContext;
+
+public abstract class AbstractCollectionValidator<A extends Annotation, E, C extends Collection<E>>
+    implements CollectionValidator<A, E, C> {
+
+  @Override
+  public boolean isValid(final C collection, final ConstraintValidatorContext context) {
+    Optional<C> items = Optional.ofNullable(collection);
+    boolean valid = true;
```

Comment:
This variable is not needed. "return false" in the loop and "return true" at the end.

Reply:
http://www.cs.utexas.edu/users/EWD/ewd02xx/EWD215.PDF